v2/orders/order-processing.rst

```diff
@@ -4,3 +4,64 @@ Order Processing
 Order processing is part of the order refresh process. This is run when
 on draft orders to ensure that it has up to date adjustments and that
 its order items are up to date.
+
+In Commerce Core, the order process moves through the following cycle:
+
+.. image:: ../images/order_workflow.png
```

Comment:
Good point. I think Shabana is basing this off our urban hipster demo which is not a default workflow.